test/nodetool/rest_api_mock.py

```diff
@@ -201,7 +172,51 @@ async def run_server(ip, port):
+            """
+            try:
+                ret = await handler(request)
+                if ret is not None:
+                    return ret
+                return aiohttp.web.Response()
+            except Exception as e:
+                tb = traceback.format_exc()
+                logger.error(f'Exception when executing {handler.__name__}: {e}\n{tb}')
+                return aiohttp.web.Response(status=500, text=str(e))
+        return catching_handler
+
+    app.router.add_routes([
+        aiohttp.web.get(f"/{server.EXPECTED_REQUESTS_PATH}", wrap_handler(server.get_expected_requests)),
+        aiohttp.web.post(f"/{server.EXPECTED_REQUESTS_PATH}", wrap_handler(server.post_expected_requests)),
+        aiohttp.web.delete(f"/{server.EXPECTED_REQUESTS_PATH}", wrap_handler(server.delete_expected_requests)),
+        # Register all required rest API paths.
+        # Unfortunately, we have to register here all the different routes, used by tests.
+        # Fortunately, aiohttp supports variable paths and with that, there is not that many paths to register.
+        aiohttp.web.route("*", "/column_family/", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/column_family/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/column_family/{part1}/{part2}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/column_family/{part1}/{part2}/{part3}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/column_family/{part1}/{part2}/{part3}/{part4}",
+                          wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/compaction_manager/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/compaction_manager/{part1}/{part2}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/failure_detector/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/gossiper/{part1}/{part2}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/snitch/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_proxy/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_service/{part1}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_service/{part1}/", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_service/{part1}/{part2}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/storage_service/{part1}/{part2}/{part3}", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/stream_manager/", wrap_handler(server.handle_generic_request)),
+        aiohttp.web.route("*", "/system/{part1}/{part2}", wrap_handler(server.handle_generic_request)),
```

Comment:
might want to add a route for `"/system/{part1}"`. as it matches with `"/system/uptime_ms"`. which is used by the "info" operaton.

Reply:
Yes, this PR needs an update after every other merged PR, that adds new routes.

Reply:
Fixed in v3.